templates/index.html

```diff
@@ -1,8 +1,12 @@
 {% extends "base.html" %}
 
-{% block title%}
-  {{ config.title }}
-{% endblock title %}
+{% block meta_content %}
+  <title>{{ config.title }}</title>
```

Comment:
FYI, based on how you've done this, when loading `https://limeleaf.io/services` you'll see a title of:
`Our Software Development Services`
and a description of the same value. There's nothing to tell it's Limeleaf. Same with the opengraph meta. You might want to add the site name/title in before the `config.title`

Reply:
I misread or got confused with how this is working and what template I was looking at. Disregard. I see in the page template there's `page.title` and `config.title`